react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -161,7 +167,7 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
         } = this.props;
         const content = _showConnectionInfo
             ? <ConnectionIndicatorContent participantId = { participantID } />
-            : this._renderRemoteVideoMenu();
+            : _disabled ? null : this._renderRemoteVideoMenu();
```

Comment:
Can you pl rewrite this as if-then-else? Nested ternary operators tend to get unwieldy